packages/cli/src/commands/serve.ts

```diff
@@ -8,6 +8,14 @@ import { toDuration, verbose } from '../common.js';
 import { logger } from '../log.js';
 import { FileTree } from '../util/file.tree.js';
 
+function rewriteFile(fileName: string, buf: ArrayBuffer, args: { baseUrl: string }): Buffer | ArrayBuffer {
+  if (fileName === 'WMTSCapabilities.xml' || fileName.endsWith('/WMTSCapabilities.xml')) {
+    const text = Buffer.from(buf).toString();
+    return Buffer.from(text.replace(/template="\//g, `template="${args.baseUrl}/v1/file/`));
```

Comment:
Is this rewrite could broke the index?

Reply:
I am not sure what you mean by that?

WMTS needs to be absolute, so this rewrites `="/tiles/0/0/0.webp"`  to `="http://localhost:8080/v1/file/tiles/0/0/0.webp"` which makes it a "valid" WMTS link

Reply:
Ahh, this is server, I thought it is the tar builder, and we need to force rewrite this file once we find it.